src/pub/ws.js

```diff
@@ -1,14 +1,15 @@
 // LICENSE_CODE ZON ISC
 'use strict'; /*jslint browser:true, react:true, es6:true*/
 import {EventTarget} from 'event-target-shim';
+import { default_ports } from './util';
 import setdb from '../../util/setdb.js';
 
 class Ws_wrapper extends EventTarget {
     constructor(){
         super();
         this.url = location.hostname;
-        this.port = location.port||80;
```

Comment:
@barakluminati here, when no has port, and the protocol is ssl, the deafault port is 443 and no 80